### PR TITLE
API calls take an optional account ID

### PR DESCRIPTION
**Pull request by mroloux**

### CODE REVIEW

File: src/test/java/seatsio/SeatsioClientTest.java [hunk 1]
```diff
@@ -52,12 +53,14 @@ protected String createTestChartWithSections() {
...
-    protected String createTestChartWithErrors() { return createTestChart("/sampleChartWithErrors.json"); }
+    protected String createTestChartWithErrors() {
+        return createTestChart("/sampleChartWithErrors.json");
+    }
 
     protected String createTestChart(String fileName) {
         String testChartJson = testChartJson(fileName);
         String chartKey = randomUUID().toString();
-        stringResponse(post(BASE_URL + "/system/public/" + user.designerKey + "/charts/" + chartKey)
+        stringResponse(Unirest.post(STAGING_BASE_URL + "/system/public/" + user.designerKey + "/charts/" + chartKey)
```
Comment (nahue) [inline src/test/java/seatsio/SeatsioClientTest.java]: Shouldn't this use UnirestUtil.post ?

File: src/main/java/seatsio/util/UnirestUtil.java [hunk 2]
```diff
@@ -32,4 +36,25 @@
             throw new SeatsioException(e, request.getHttpRequest());
         }
     }
+
+    public static GetRequest get(String url, String secretKey, Long accountId) {
+        return authenticate(Unirest.get(url), secretKey, accountId);
+    }
+
+    public static HttpRequestWithBody post(String url, String secretKey, Long accountId) {
+        return authenticate(Unirest.post(url), secretKey, accountId);
```
Comment (nahue) [inline src/main/java/seatsio/util/UnirestUtil.java]: UnirestUtil ?

Comment (mroloux) [reply src/main/java/seatsio/util/UnirestUtil.java]: The class `UnirestUtil` uses `Unirest` under the hood

File: src/test/java/seatsio/SeatsioClientTest.java (hunk 1 above)
Comment (mroloux) [reply src/test/java/seatsio/SeatsioClientTest.java]: `UnirestUtil.post` sends a basic auth header, but that's not needed here. The `/system/public` URLs contain a public key for authentication.

**Pull request merged**